.gitlab/ci/global.yml

```
@@ -13,67 +13,69 @@
+  - chmod +x ./Tests/scripts/*
+  - chmod +x ./Tests/Marketplace/*
+  - section_end "Granting execute permissions on files"
 
-    - section_start "Installing Virtualenv" --collapsed
-    - |
-      if [ -f "./venv/bin/activate" ]; then
-        echo "found venv"
-      else
-        echo "installing venv"
-        NO_HOOKS=1 SETUP_PY2=yes .hooks/bootstrap >> $ARTIFACTS_FOLDER/logs/installations.log 2>&1
-      fi
-    - venv/bin/pip3 install -r .circleci/build-requirements.txt >> $ARTIFACTS_FOLDER/logs/installations.log 2>&1
-    - source ./venv/bin/activate
-    - |
-      if [ -n "${DEMISTO_SDK_NIGHTLY}" ]; then
-        echo "Installing SDK from master branch" | tee --append $ARTIFACTS_FOLDER/logs/installations.log
-        pip3 install git+https://github.com/demisto/demisto-sdk@master >> $ARTIFACTS_FOLDER/logs/installations.log 2>&1
-      fi
-    - section_end "Installing Virtualenv"
-
-    - section_start "Installing SSH keys" --collapsed
-    - eval $(ssh-agent -s)
-    - chmod 400 $OREGON_CI_KEY
-    - ssh-add $OREGON_CI_KEY
-    - mkdir -p ~/.ssh
-    - chmod 700 ~/.ssh
-    - section_end "Installing SSH keys"
-
-    - section_start "Build Parameters"
-    - set | grep -E "^NIGHTLY=|^INSTANCE_TESTS=|^SERVER_BRANCH_NAME=|^ARTIFACT_BUILD_NUM=|^DEMISTO_SDK_NIGHTLY=|^TIME_TO_LIVE=|^CONTRIB_BRANCH=|^FORCE_PACK_UPLOAD=|^PACKS_TO_UPLOAD=|^BUCKET_UPLOAD=|^GCS_MARKET_BUCKET=|^SLACK_CHANNEL="
-    - python --version
-    - python3 --version
-    - node --version
-    - npm --version
-    - demisto-sdk --version
-    - section_end "Build Parameters"
+  - section_start "Installing Virtualenv" --collapsed
+  - |
+    if [ -f "./venv/bin/activate" ]; then
+      echo "found venv"
+    else
+      echo "installing venv"
+      NO_HOOKS=1 SETUP_PY2=yes .hooks/bootstrap >> $ARTIFACTS_FOLDER/logs/installations.log 2>&1
+    fi
+  - venv/bin/pip3 install -r .circleci/build-requirements.txt >> $ARTIFACTS_FOLDER/logs/installations.log 2>&1
+  - source ./venv/bin/activate
+  - |
+    if [ -n "${DEMISTO_SDK_NIGHTLY}" ]; then
+      echo "Installing SDK from master branch" | tee --append $ARTIFACTS_FOLDER/logs/installations.log
+      pip3 install git+https://github.com/demisto/demisto-sdk@master >> $ARTIFACTS_FOLDER/logs/installations.log 2>&1
+    fi
+  - section_end "Installing Virtualenv"
+
+  - section_start "Installing SSH keys" --collapsed
+  - eval $(ssh-agent -s)
+  - chmod 400 $OREGON_CI_KEY
+  - ssh-add $OREGON_CI_KEY
+  - mkdir -p ~/.ssh
+  - chmod 700 ~/.ssh
+  - section_end "Installing SSH keys"
+
+  - section_start "Build Parameters"
+  - set | grep -E "^NIGHTLY=|^INSTANCE_TESTS=|^SERVER_BRANCH_NAME=|^ARTIFACT_BUILD_NUM=|^DEMISTO_SDK_NIGHTLY=|^TIME_TO_LIVE=|^CONTRIB_BRANCH=|^FORCE_PACK_UPLOAD=|^PACKS_TO_UPLOAD=|^BUCKET_UPLOAD=|^GCS_MARKET_BUCKET=|^SLACK_CHANNEL="
+  - python --version
+  - python3 --version
+  - node --version
+  - npm --version
+  - demisto-sdk --version
+  - section_end "Build Parameters"
+
+.default-before_script:
```

```suggestion
.default-before-script:
```
Let's be consistent with our naming conventions. I think all of our other hidden job configurations use dashes in the name.